exaudfclient/base/javacontainer/script_options/test/converter_test.cc

```diff
@@ -7,22 +7,24 @@
 using namespace SWIGVMContainers::JavaScriptOptions;
 
 
-class LegacyConverterJarTest : public ::testing::TestWithParam<std::pair<std::string, std::set<std::string>>> {};
+class LegacyConverterJarTest : public ::testing::TestWithParam<std::pair<std::string, std::vector<std::string>>> {};
 
 TEST_P(LegacyConverterJarTest, jar) {
-    const std::pair<std::string, std::set<std::string>> option_value = GetParam();
+    const std::pair<std::string, std::vector<std::string>> option_value = GetParam();
     const std::string jar_option_value = option_value.first;
 
     ConverterLegacy converter;
     converter.convertExternalJar(option_value.first);
-    ASSERT_EQ(converter.getJarPaths(), option_value.second);
+    std::vector<std::string> result;
+    converter.iterateJarPaths([&](auto jar) {result.push_back(jar);});
+    ASSERT_EQ(result, option_value.second);
 }
 
-const std::vector<std::pair<std::string, std::set<std::string>>> jar_strings =
+const std::vector<std::pair<std::string, std::vector<std::string>>> jar_strings =
         {
-            std::make_pair("test.jar:test2.jar", std::set<std::string>({"test.jar", "test2.jar"})),
-            std::make_pair("\"test.jar:test2.jar\"", std::set<std::string>({"\"test.jar", "test2.jar\""})),
-            std::make_pair("t\\:est.jar:test2.jar", std::set<std::string>({"t\\", "est.jar", "test2.jar"})),
```

Comment:
The new values are not equivalent complex

Reply:
I added comments to the test data to explain it's purpose